src/Trixi.jl

```diff
@@ -75,26 +75,27 @@ export flux_central, flux_lax_friedrichs, flux_hll, flux_upwind,
        flux_chandrashekar, flux_ranocha, flux_derigs_etal, flux_kennedy_gruber, flux_shima_etal
 
 # TODO: Taal decide, which initial conditions and source terms will be used/exported
-export initial_conditions_convergence_test,
-       initial_conditions_gauss,
-       initial_conditions_weak_blast_wave, initial_conditions_blast_wave,
-       initial_conditions_khi,
-       initial_conditions_isentropic_vortex, initial_conditions_blob,
-       initial_conditions_density_wave, initial_conditions_sedov_blast_wave,
-       initial_conditions_orszag_tang, initial_conditions_rotor
+export initial_condition_convergence_test,
+       initial_condition_gauss,
+       initial_condition_weak_blast_wave, initial_condition_blast_wave,
+       initial_condition_khi,
+       initial_condition_isentropic_vortex, initial_condition_blob,
+       initial_condition_density_wave, initial_condition_sedov_blast_wave,
+       initial_condition_orszag_tang, initial_condition_rotor
 
-export source_terms_convergence_test, source_terms_harmonic
+export boundary_condition_convergence_test, boundary_condition_gauss
 
-export boundary_conditions_convergence_test, boundary_conditions_gauss
+export source_terms_convergence_test, source_terms_harmonic
 
 export TreeMesh
 
 export DG,
        DGSEM, LobattoLegendreBasis,
        VolumeIntegralWeakForm, VolumeIntegralFluxDifferencing,
-       VolumeIntegralShockCapturingHG, IndicatorHennemannGassner
+       VolumeIntegralShockCapturingHG, IndicatorHennemannGassner,
+       MortarL2
 
-export SemidiscretizationHyperbolic, semidiscretize, compute_coefficients
+export SemidiscretizationHyperbolic, semidiscretize, compute_coefficients, integrate
```

Comment:
Exporting something usually means that we consider it to be a part of our API. Since we weren't completely happy with the name `integrate_via_indices`, I thought it might be better to not export than function now. If you think we'll stick with the name `integrate_via_indices`, we can of course export it from now on.

Reply:
No, you're right. Good point.